test/metabase/api/user_test.clj

```diff
@@ -326,9 +329,19 @@
 ;;; |                                      Updating a User -- PUT /api/user/:id                                      |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
+(defn include-personal-collection-name
+  {:hydrate :personal_collection_name}
+  [user]
+  (db/select-one-field
+      :name Collection :id (:personal_collection_id user)))
```

Comment:
According to the [Clojure Style Guide](https://guide.clojure.style/#one-space-indent) this line should be indented with one space only. However, in this case I would suggest putting the arguments on the same line as the function call, consistent with how it's used in the rest of the file.

Sorry I missed this before. This is the last change, I'm sure! :stuck_out_tongue_winking_eye: